ecl/eclagent/eclagent.cpp

```diff
@@ -3199,9 +3180,8 @@ char * EclAgent::getDaliServers()
     return dali.detach();
 }
 
-void EclAgent::addTimings()
+void EclAgent::addTimings(WorkunitUpdate w)
```

Comment:
Probably better to have an IWorkUnit * parameter (no other code should need to change).  (Otherwise a function passing an IWorkUnit * would be likely to have the parameter freed by accident.)